CHANGELOG.md

```diff
@@ -51,6 +51,14 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
 - `cmd/mdatagen`: Remove `level` field from metrics definition (#12145)
   This mechanism will be added back once a new views mechanism is implemented.
 - `service`: Value for telemetry exporter `otlp.protocol` updated from `grpc/protobuf` to `grpc`. (#12337)
+- `service`: internal metrics exported over Prometheus may differ from previous versions. (#11611)
+
+  The change to update the internal telemetry to use [otel-go config](https://pkg.go.dev/go.opentelemetry.io/contrib/config) can cause unexpected behaviour
+  for end users. This change is caused by the default values in `config` being different from what the Collector has used in previous versions. The
+  following changes can occur when users configure their `service::telemetry::metrics::readers`:
```

Comment:
QQ: does it mean as long as you don't explicitly configure `readers` you won't run into this? e.g. if I just have
```
service:
  telemetry:
    metrics:
      level: detailed
```

the behavior of this should be the same in all versions, right?

Reply:
Yes, I think it worth mentioning that the breaking change only affects user how customized the prometheus reader in any way. The default behavior is the same

Reply:
Tried to add a note at the beginning to capture this, ptal @dmitryax @songy23